app/models/solid_queue/job/executable.rb

```diff
@@ -79,7 +75,6 @@ def ready
           ReadyExecution.create_or_find_by!(job_id: id)
         end
 
-
```

Comment:
This new line was on purpose 😊 It adds a visual separation between methods related to each other, for enqueuing the job and a method for a setting not directly related to the 3 methods above.

Reply:
Gotcha. Haven't seen that style before. I'll revert.